server/embed/config.go

```diff
@@ -317,6 +317,7 @@ type Config struct {
 	// ExperimentalEnableLeaseCheckpoint enables primary lessor to persist lease remainingTTL to prevent indefinite auto-renewal of long lived leases.
 	ExperimentalEnableLeaseCheckpoint       bool          `json:"experimental-enable-lease-checkpoint"`
 	ExperimentalCompactionBatchLimit        int           `json:"experimental-compaction-batch-limit"`
+	ExperimentalCompactionSleepInterval     int           `json:"experimental-compaction-sleep-interval"`
```

Comment:
time.Duration ?

Please add a comment describing what is it.

Reply:
done